lib/mayaUsd/resources/scripts/mayaUsdAddMayaReference.py

```diff
@@ -49,47 +53,158 @@ def createPrimAndAttributes(stage, primPath, mayaReferencePath, mayaNamespace, m
 
     return prim
 
+def getDefaultGroupPrimName(parentPrim, mayaNamespace):
+    # "The name of the reference file" + "RN" + "group" for example 'sphereRNgroup'.
+    # If a prim with this name already exists, then add a numeric suffix after "RN".
+    startName = mayaNamespace + "RNgroup"
+    checkName = mayaUsd.ufe.uniqueChildName(parentPrim, startName)
+    index = 1
+    while (checkName != startName):
+        startName = "%sRN%dgroup" % (mayaNamespace, index)
+        checkName = mayaUsd.ufe.uniqueChildName(parentPrim, startName)
+        index += 1
+    return startName
```

Comment:
Cannot use "uniqueChildName" directly since the **index** to increment for the group prim name is inside the name and not at the end.